.github/workflows/build.yaml

```diff
@@ -0,0 +1,56 @@
+name: CI Build & Push
+on:
+  pull_request_target:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  build-and-push-prs:
+    timeout-minutes: 360
+    name: Build and push multi-arch images
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Add swap
+        run: |
+          set -x
+          file="/var/lib/swap"
+          sudo fallocate -l 4G "${file}"
+          sudo chmod 600 "${file}"
+          sudo mkswap "${file}"
+          sudo swapon "${file}"
+          sudo swapon -s
+          free -m
+          df -h
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/buildx-cache
+          key: docker-cache-${{ github.head_ref }}
+          restore-keys: docker-cache-master
+      - name: Login to quay.io
+        uses: docker/login-action@v1
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_PASSWORD }}
+      - name: Checkout PR
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Prep for build
+        run: echo "${{ github.event.pull_request.head.sha }}" >SOURCE_VERSION
+      - name: PR Multi-arch build & push of cilium-envoy
+        uses: docker/build-push-action@v2
+        id: docker_build_ci
+        with:
+          context: .
+          file: ./Dockerfile
+          platforms: linux/amd64,linux/arm64
+          build-args: |
+            BUILDER_IMAGE=quay.io/cilium/cilium-envoy-builder:release-archive-latest
+            BAZEL_BUILD_OPTS=--remote_upload_local_results=false
+          cache-from: type=local,src=/tmp/buildx-cache
+          cache-to: type=local,dest=/tmp/buildx-cache,mode=max
+          push: true
+          tags: quay.io/${{ github.repository_owner }}/cilium-envoy-dev:${{ github.event.pull_request.head.sha }}
```

Comment:
I would just add a final step to print the SHA of the images created. Something like
```yaml

      - name: Image Release Digest
        shell: bash
        run:
          echo "Digests:"
          echo "quay.io/${{ github.repository_owner }}/cilium-envoy-dev:${{ github.event.pull_request.head.sha }}@${{ steps.docker_build_ci.outputs.digest }}"
```

Reply:
Added, testing again to make sure I did not mess this up :-)